src/app/space/create/deployments/deployments-linechart/deployments-linechart.component.ts

```diff
@@ -0,0 +1,179 @@
+import {
+  Component,
+  DoCheck,
+  Input,
+  OnInit,
+  ViewEncapsulation
+} from '@angular/core';
+
+import {
+  ChartBase,
+  ChartDefaults
+} from 'patternfly-ng';
+
+import {
+  cloneDeep,
+  defaultsDeep,
+  isEqual,
+  merge,
+  uniqueId
+} from 'lodash';
+
+import { DeploymentsLinechartConfig } from './deployments-linechart-config';
+import { DeploymentsLinechartData } from './deployments-linechart-data';
+
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'deployments-linechart',
```

Comment:
Please add the f8- namespace prefix to the selector.

Reply:
Done. When are we expected to use this convention for selectors? Should it always be done?